File: FormalBench-data/diverse/rule_21/CountOddSquares.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class CountOddSquares {

	public static int countOddSquares(int n, int m) {
		int count = 0;
		for (int i = n; i <= m; i++) {
			double square = Math.sqrt(i);
			if (0 == square % 1 && 0 == i % 1) {
				count++;
			}
		}
		return count;
	}
}

File: FormalBench-data/FilteredData/translated_c/seed_726_per_category_10_653ade686f/CountOddSquares.c
#include "java_arrays.h"
#include <math.h>
#include <stdint.h>

static int32_t java_add(int32_t left, int32_t right) {
    return (int32_t)((uint32_t)left + (uint32_t)right);
}

int32_t countOddSquares(int32_t n, int32_t m) {
    int32_t count = 0;
    for (int32_t i = n; i <= m; i = java_add(i, 1)) {
        double square = sqrt((double)i);
        if (fmod(square, 1.0) == 0.0 && i % 1 == 0) {
            count = java_add(count, 1);
        }
    }
    return count;
}
